bigquery/bigquery_create_object_table/main.tf

```diff
@@ -0,0 +1,74 @@
+/**
+* Copyright 2023 Google LLC
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
```

Comment:
Not required but do consider adding a comment at the top of the sample with a brief synopsis. Example: https://github.com/terraform-google-modules/terraform-docs-samples/blob/c0e42baa72e0961543d31fa6fcfc8de8fb4cf3b6/run/secret_manager/main.tf#L17

Reply:
done. added doc.